### PR TITLE
"Customize Gym" feature implemented

### PR DESCRIPTION
**Pull request by jiaweini**

### CODE REVIEW

File: app/src/main/res/layout/activity_homeequipment.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        android:layout_marginLeft="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:layout_marginBottom="40dp"
+        android:onClick="onSubmitEquipmentClick"
+        android:text="Submit equipment"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.497"
+        app:layout_constraintStart_toStartOf="parent" />
```
Comment (bendwilletts) [inline app/src/main/res/layout/activity_homeequipment.xml]: Button position is covering the list of equipment. I would add the following line of code.

`app:layout_constraintTop_toBottomOf="@id/equipmentListView"`

File: app/src/main/res/layout/activity_homeequipment.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    <ListView
+        android:id="@+id/equipmentListView"
+
+        android:layout_width="309dp"
+        android:layout_height="469dp"
```
Comment (bendwilletts) [inline app/src/main/res/layout/activity_homeequipment.xml]: change to `android:layout_height="0dp"`

File: app/src/main/res/layout/activity_homeequipment.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginTop="60dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintBottom_toBottomOf="parent"
```
Comment (bendwilletts) [inline app/src/main/res/layout/activity_homeequipment.xml]: change to `app:layout_constraintBottom_toTopOf="@id/submitEquipmentBtn"`

**Review by bendwilletts — CHANGES_REQUESTED** [review]
Looks great to me other than a minor UI issue. The submit button covers the ListView, I commented a few one line changes that fixed it on my end. Aside from this, should be good to merge after fixing.

**Comment by jiaweini** [comment]
The UI changes are made.

**Review by bendwilletts — APPROVED** [review]

**Comment by bendwilletts** [comment]
looks great, merge when ready

**Pull request merged**